internal/pkg/agent/application/paths/common.go

```diff
@@ -46,14 +47,23 @@ func init() {
 	logsPath = topPath
 	unversionedHome = false // only versioned by container subcommand
 
+	// these should never change
```

Comment:
Could you explain why? For example, mentioning that components are bundled with the agent binary and we don't download anything anymore.

It might specifically be worth mentioning that these directories should not be moved to any newly configured state path.